transport/nio/src/test/java/org/kaazing/gateway/transport/tcp/specification/TcpAcceptorRule.java

```diff
@@ -127,4 +127,8 @@ public TcpAcceptorRule addConfigurationProperty(String propertyName, String prop
         return this;
     }
 
+    public ResourceAddressFactory getAddressFactory() {
+        return addressFactory;
+    }
+
```

Comment:
It was done by David when we initiated the tcp ip whitelist support, so I'm not sure why it was done, but I will change it back. Let me know if I should also add the test he removed.